testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/managedbean/gc/CountBean.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.as.test.integration.jsf.managedbean.gc;
```

Comment:
Just minor, this file and a few others are missing copyright headers.

Reply:
Added headers